packages/clerk-js/src/ui/components/UserButton/useMultisessionActions.tsx

```diff
@@ -41,11 +41,27 @@ export const useMultisessionActions = (opts: UseMultisessionActionsParams) => {
         })();
       });
     }
-
     openUserProfile(opts.userProfileProps);
     return opts.actionCompleteCallback?.();
   };
 
+  const handleUserProfileActionClicked = (path?: string) => {
+    if (opts.userProfileMode === 'navigation') {
+      return navigate(opts.userProfileUrl || '').finally(() => {
+        void (async () => {
+          await sleep(300);
+          opts.actionCompleteCallback?.();
+        })();
+      });
+    }
+    openUserProfile({
+      ...opts.userProfileProps,
+      ...(path && { path }),
```

Comment:
@anagstef @LekoArts @octoper Do you think we should reuse the existing `path` prop in our modals ?
I'm suggesting `startPath` or `initialPath`, the current `path` prop is very tight to our "mounted components". WDYT ?

Reply:
When we land on the name I think we should prefix it with `__experimental_` until we ship this official as part of Custom Items.

Reply:
`startPath` seems fine, we use it in other places, too